docs/pages/aggregator.mdx

```diff
@@ -0,0 +1,173 @@
+# ERC7786 Aggregator
+
+## Why does it matter?
+
+When someone sends a cross-chain message using a given protocol, they're relying on its security, robustness and speed. This is ok for most retail users and use cases, but for some use cases, it might be necessary to add extra layers of security while handling with cross-chain messages.
```

Comment:
```suggestion
When someone sends a cross-chain message using a given protocol, they're relying on its security, robustness and speed. This is ok for most retail users and use cases, but in other cases it might be necessary to add extra layers of security while handling with cross-chain messages.
```